indico/modules/categories/controllers/display.py

```diff
@@ -97,3 +102,42 @@ def _process_stats(self, stats, root=False):
 
     def _count_users(self):
         return User.find(is_deleted=False, is_pending=False).count()
+
+
+def _serialize_category(category, with_path=False):
+    data = {
+        'id': category.id,
+        'title': category.title,
+        'is_protected': category.is_protected,
+        'category_count': category.deep_children_count,
+        'event_count': category.deep_events_count,
+        'can_access': category.can_access(session.user)
+    }
+    if with_path:
+        data['path'] = [{'id': c.id, 'title': c.title} for c in category.parent_chain_query]
+    return data
+
+
+class RHCategoryInfo(RHDisplayCategoryBase):
+
```

Comment:
there's no empty line here unless there's a docstring (in that case there's an empty line *after* the docstring)